codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -385,7 +389,7 @@ private ChannelFuture writeGoAway(ChannelHandlerContext ctx, Http2Exception caus
         // the process.
```

Comment:
1. Remove the redundant if check at the top of this method (now out of date).
2. Fix the spelling on the comment on the line above `alredy` -> `already`.
3. Move this comment up onto the previous line.
4. Thank you :)